### PR TITLE
Newq jsquestion8

### PR DESCRIPTION
**Pull request by kunalkr789**
PR comments resolution

### CODE REVIEW

**Review by shivamyth — APPROVED** [review]

File: javascript/answers/kunalkr789-q8.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What are escape Sequences?
```
Comment (arpansac) [inline javascript/answers/kunalkr789-q8.html]: Change this to the question text!

**Review by arpansac — CHANGES_REQUESTED** [review]
Just one small change

File: javascript/answers/kunalkr789-q8.html (hunk 1 above)
Comment (kunalkr789) [reply javascript/answers/kunalkr789-q8.html]: change is done

**Review by dakshkhetan — APPROVED** [review]
Looks good!

**Pull request merged**